File: PHP PowerPoint Examples/DocumentPropertiesApiDemo/getPptDocumentProperties.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\PropertiesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new PropertiesApi($configuration);

$name = "Properties.pptx";
$password = null;
$folder = "input";
$storage = null;
$result = $apiInstance->getPptDocumentProperties($name, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/PropertiesApiDemo/getPptDocumentProperties.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.properties_api.PropertiesApi(configuration)

name = "Properties.pptx" #Supported formats:ppt/pptx/pptm/pps/ppsx/pot/potx/odp
password = ""
storage = ""
folder = "input"

result = api.get_ppt_document_properties(name, password=password, storage=storage, folder=folder)